index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
+        [=claims=] made by an [=issuer=] about a [=subject=]. Examples include
```

Comment:
VCDM says multiple subjects are allowed in a credential.

Reply:
```suggestion
        [=claims=] made by an [=issuer=] about one or more [=subjects=]. Examples include
```

Reply:
Fixed in 002d8ae2eb75b88d60b2e7d3b44ac5c05ddcf397